packages/react-radio/src/stories/RadioGroupDefaultValue.stories.tsx

```diff
@@ -0,0 +1,23 @@
+import * as React from 'react';
+import { Radio, RadioGroup } from '../index';
+
+export const DefaultValue = () => {
```

Comment:
What do you think about calling this story `RadioGroupUncontrolledValue`? The controlled version of this story is called `RadioGroupControlledValue` and my feeling (and limited experience from Shield) is that people see "default" and think "this is the way I should use this component" when really uncontrolled vs controlled inputs is up to the user (or, if anything, [controlled inputs should be considered default in React](https://reactjs.org/docs/uncontrolled-components.html))